### ISSUE TITLE
Switch to Azure Entra External ID

### ISSUE THREAD

**Issue by leo-schick** [Azure/azure-saas#263]
A few month back, Azure Entra has been announced and Azure Entra External ID is in preview. Is there a plan to switch this project over to use Azure Entra External ID? Or is there an alternative SAAS toolkit in planning which will integrate with Entra External ID?

**Comment by 1iveowl** [comment]
Agree. This would be a natural evolution of the toolkit. Preliminary work have begun to update this toolkit. There’s no release date for an update at the moment. Also, as you mention, key part of Entra is in preview at the moment.
